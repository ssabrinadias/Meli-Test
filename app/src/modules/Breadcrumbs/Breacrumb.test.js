import React from "react";
import { render} from '@testing-library/react'
import Breadcrumb from ".";

describe("Breadcrumb component", () => {
	it("[snapshot] - should be able to render ", () => {
		const { container } = render(<Breadcrumb />);
		expect(container).toMatchSnapshot();
	});
});
