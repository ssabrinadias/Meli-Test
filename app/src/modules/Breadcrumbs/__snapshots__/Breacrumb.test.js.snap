// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb component [snapshot] - should be able to render  1`] = `
<div>
  <div
    class="sc-AxjAm dJLMUN"
  >
    <ul
      class="sc-AxirZ aGDiX"
    />
  </div>
</div>
`;
